<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>appinfo</class>
 <widget class="QWidget" name="appinfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
width: 144px;
height: 50px;
opacity: 1;
font-size: 36px;
font-family: Roboto, Roboto-Bold;
font-weight: Bold;
text-align: left;
color: #556f59;
line-height: 0px;
}</string>
        </property>
        <property name="text">
         <string>车辆信息</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
width: 500px;
height: 64px;
opacity: 1;
background: #556f59;
border: 1px solid #777777;
}</string>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>30</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
width: 54px;
height: 50px;
opacity: 1;
font-size: 18px;
font-family: Microsoft YaHei, Microsoft YaHei-Bold;
font-weight: Bold;
text-align: left;
color: #fafafa;
line-height: 50px;
border-width:0;
border-style:outset;
background-color:transparent;
}</string>
         </property>
         <property name="text">
          <string>车牌号：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>30</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
width: 191px;
height: 39px;
opacity: 1;
font-size: 28px;
font-family: Microsoft YaHei, Microsoft YaHei-Bold;
font-weight: Bold;
text-align: left;
color: #fafafa;
line-height: 0px;
border-width:0;
border-style:outset;
background-color:transparent;

}</string>
         </property>
         <property name="text">
          <string>鄂A 0D123</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
width: 500px;
height: 64px;
opacity: 1;
background: #556f59;
border: 1px solid #777777;
}</string>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>30</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
width: 54px;
height: 50px;
opacity: 1;
font-size: 18px;
font-family: Microsoft YaHei, Microsoft YaHei-Bold;
font-weight: Bold;
text-align: left;
color: #fafafa;
line-height: 50px;
border-width:0;
border-style:outset;
background-color:transparent;
}</string>
         </property>
         <property name="text">
          <string>车 主：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>30</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
width: 191px;
height: 39px;
opacity: 1;
font-size: 28px;
font-family: Microsoft YaHei, Microsoft YaHei-Bold;
font-weight: Bold;
text-align: left;
color: #fafafa;
line-height: 0px;
border-width:0;
border-style:outset;
background-color:transparent;

}</string>
         </property>
         <property name="text">
          <string>张师傅</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
width: 500px;
height: 64px;
opacity: 1;
background: #556f59;
border: 1px solid #777777;
}</string>
        </property>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>30</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
width: 54px;
height: 50px;
opacity: 1;
font-size: 18px;
font-family: Microsoft YaHei, Microsoft YaHei-Bold;
font-weight: Bold;
text-align: left;
color: #fafafa;
line-height: 50px;
border-width:0;
border-style:outset;
background-color:transparent;
}</string>
         </property>
         <property name="text">
          <string>车 型：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>20</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
width: 191px;
height: 39px;
opacity: 1;
font-size: 28px;
font-family: Microsoft YaHei, Microsoft YaHei-Bold;
font-weight: Bold;
text-align: left;
color: #fafafa;
line-height: 0px;
border-width:0;
border-style:outset;
background-color:transparent;

}</string>
         </property>
         <property name="text">
          <string>白色</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>20</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
width: 191px;
height: 39px;
opacity: 1;
font-size: 28px;
font-family: Microsoft YaHei, Microsoft YaHei-Bold;
font-weight: Bold;
text-align: left;
color: #fafafa;
line-height: 0px;
border-width:0;
border-style:outset;
background-color:transparent;

}</string>
         </property>
         <property name="text">
          <string>凯迪拉克</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
width: 438px;
height: 30px;
opacity: 1;
font-size: 21px;
font-family: Microsoft YaHei, Microsoft YaHei-Bold;
font-weight: Bold;
text-align: left;
color: #ffa300;
line-height: 0px;
}</string>
        </property>
        <property name="text">
         <string>您呼叫的车辆正在赶来，请尽快赶往乘车地点！</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
